frontend/src/components/InputField/InputField.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useField } from 'formik';
+import { InputGroup, FormControl } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import styles from './styles.module.sass';
+
+interface IProps {
+    label: string;
+    name: string;
+    type: string;
+    placeholder?: string;
+    link?: string;
+    linkDescription?: string;
+    linkClassName?: string;
+}
+
+const InputField = ({
```

Comment:
You forgot to set a type. `const InputField: React.FC(or FunctionComponent)<IProps>`
Also, the name of the file should be index.tsx